### PR TITLE
More on semiT2

### PR DESCRIPTION
**Pull request by StevenClontz**
Reimplementation of #421 but rebased on main

### CODE REVIEW

**Comment by StevenClontz** [comment]
Establishes examples for

- https://topology.pi-base.org/spaces?q=US%2B~Semi-Hausdorff
- https://topology.pi-base.org/spaces?q=~US%2BSemi-Hausdorff

**Comment by StevenClontz** [comment]
Disclosure: this is all original research from a preprint I'm working on. So the question is whether I should move some of this work to another forum before approval.

**Comment by prabau** [comment]
Interesting example with S185.  It seems one should be able to show it's semiregular and also totally disconnected, similar to S97.  Then semi-Hausdorff can be deduced.

File: spaces/S000185/README.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+uid: S000185
+name: Compact semi-Hausdroff non-KC space
```
Comment (prabau) [inline spaces/S000185/README.md:3]: typo: "semi-Hausdorff"

**Comment by prabau** [comment]
For locally metrizable: shouldn't the open metric fan in $\mathbb R^2$ also contain the origin $(0,0)$, which would correspond to $\infty_x$ or $\infty_y$?

**Comment by prabau** [comment]
Sorry, noticed it's back to draft.  Not approving for now.

**Comment by StevenClontz** [comment]
The draft may be an error - I thought I'd changed it back...

**Comment by StevenClontz** [comment]
Ah, didn't quite finish that commit adding totally disconnected. Pushed and now ready.

**Comment by prabau** [comment]
Added missing braces and a closing parenthesis, plus some parentheses for readability, plus an extra word.  Please check if it's ok.

**Review by prabau — APPROVED** [review]

**Pull request merged**